java/src/main/java/ai/onnxruntime/OnnxRuntime.java

```diff
@@ -201,11 +212,11 @@ private static void load(Path tempDirectory, String library) throws IOException
             os.write(buffer, 0, readBytes);
           }
         }
-        System.load(tempFile.getAbsolutePath());
+        if (system_load) System.load(tempFile.getAbsolutePath());
```

Comment:
I think this is the only inline if statement in the Java codebase. It should probably also include the logger message, and have an else branch that has a logging message at fine saying "library extracted" or something similar.

Reply:
Yeah, spotless does things like that. Our group in Oracle Labs always keeps braces around if statements, and so the ORT Java API is like that, and I didn't want it to stick out.